src/components/application_manager/src/app_launch/app_launch_data_json.cc

```diff
@@ -219,7 +236,9 @@ bool AppLaunchDataJson::DeleteOldestAppData() {
   sync_primitives::AutoLock autolock(app_launch_json_lock_);
   std::vector<uint64_t> temp_array;
   std::vector<Json::Value> temp_json_list;
-  Json::Value& apps_list = GetSavedApplicationDataList();
+  resumption::LastStateAccessor accessor = last_state_wrapper_->get_accessor();
+  Json::Value dictionary = accessor.GetData().dictionary();
```

Comment:
@LitvinenkoIra what is the purpose of copying dictionary if it won''t be modified? Can you init is with cont ref?

Reply:
@LuxoftAKutsan we modify the dictionary in _GetApplicationData_ function .
DeleteOldestAppData -> GetSavedApplicationDataList -> GetApplicationData